src/Service/CloudWatchLogs/src/Result/DescribeLogStreamsResponse.php

```diff
@@ -123,7 +123,8 @@ protected function populateResult(Response $response): void
             }
 
             return $items;
-        })($data['logStreams']);
+        };
+        $this->logStreams = empty($data['logStreams']) ? [] : $fn['list-LogStreams']($data['logStreams']);
```

Comment:
Could you help me to understand why this changed? What is the purpose? Or was it a side-effect of something?

Reply:
The $fn construct has the purpose of handling recursion. I this file it does not change anything. But in src/Service/RdsDataService/src/Result/BatchExecuteStatementResponse.php it there are function calls within functions.

Reply:
Thank you for the explanation.